lib/RegisterController.js

```diff
@@ -17,6 +17,7 @@ export class RegisterController {
       registering: false,
       checkingExistence: false,
       email: null,
+      token: null
```

Comment:
What's this `token` for? Seems like maybe this is from a previous revision and should be `authorization` instead?

Also, setting it to `null` -- would that be a problem for the schema vs. `undefined`? I suspect this should have `authorization: undefined`.

Reply:
Removed, it was unused.